client/src/components/DataFiles/DataFilesProjectMembers/DataFilesProjectMembers.js

```diff
@@ -160,41 +171,46 @@ const DataFilesProjectMembers = ({
         Add Member
       </Label>
       <div styleName="user-search">
-        <div className="input-group">
-          <div className="input-group-prepend">
-            <Button
-              styleName="add-button member-search"
-              onClick={() => onAdd({ user: selectedUser, access: 'edit' })}
-              disabled={
-                !selectedUser ||
-                loading ||
-                alreadyMember(selectedUser) ||
-                mode === 'transfer'
-              }
-            >
-              Add
-            </Button>
-          </div>
-          <Input
-            list="user-search-list"
-            type="text"
-            onChange={e => userSearch(e)}
-            placeholder="Search by name"
-            styleName="member-search"
-            disabled={loading || mode === 'transfer'}
-          />
-          <datalist id="user-search-list">
-            {/* eslint-disable */
+        <form ref={formRef}>
+          <div className="input-group" styleName="member-search-group">
+            <div className="input-group-prepend">
+              <Button
+                styleName="add-button member-search"
+                onClick={() =>
+                  onAddCallback({ user: selectedUser, access: 'edit' })
+                }
+                disabled={
+                  !selectedUser ||
+                  loading ||
+                  alreadyMember(selectedUser) ||
+                  mode === 'transfer'
+                }
+              >
+                Add
+              </Button>
+            </div>
+            <Input
+              list="user-search-list"
+              type="text"
+              onChange={e => userSearch(e)}
+              placeholder="Search by name"
+              styleName="member-search"
+              disabled={loading || mode === 'transfer'}
+              autoComplete="false"
+            />
+            <datalist id="user-search-list">
+              {/* eslint-disable */
               // Need to replace this component with a generalized solution from FP-743
               userSearchResults
                 .filter(user => !alreadyMember(user))
                 .map(user => (
                 <option value={formatUser(user)} key={user.username} />
               ))
               /* eslint-enable */
-            }
-          </datalist>
-        </div>
+              }
+            </datalist>
+          </div>
+        </form>
```

Comment:
__Console Error__

Open "Add Shared Workspace" modal for first time since page load:
1. Reload Portal webpage.
1. Add Shared Workspace (Data Files dropdown, any sub-section).
1. __See console error:__
    ```
    Warning: validateDOMNesting(...): <form> cannot appear as a descendant of <form>.
    in form (at DataFilesProjectMembers.js:174)
    ```
1. Do __not__ reload page.
1. Add Shared Workspace (Data Files dropdown, any sub-section).
1. No console error.

It seems like `<Form>` is used in a modal, then `<form>` is used in a component loaded into the modal.